api/src/main/scala/org/ensime/api/outgoing.scala

```diff
@@ -475,11 +475,11 @@ final case class BasicTypeInfo(
 
 final case class ArrowTypeInfo(
     name: String,
+    fullName: String,
```

Comment:
Is this a fully qualified name?  - is there a better variable name or type we can use?